postgres/codenamemap.yaml

```diff
@@ -76,10 +88,20 @@
 {{ debian_codename('focal', '12') }}
 
 ## Fedora
-# http://apt.postgresql.org/pub/repos/yum/reporpms/
+# https://download.postgresql.org/pub/repos/yum/13/fedora/
 # https://apps.fedoraproject.org/packages/postgresql
-# `oscodename` grain has long distro name
-{{ fedora_codename('Fedora-31', '11', 'Fedora 31 (Thirty One)') }}
-{{ fedora_codename('Fedora-30', '11', 'Fedora 30 (Thirty)') }}
```

Comment:
Ever since building the pre-salted images with the `salt-image-builder`, we've found that the `oscodename` grain has been empty on Fedora since version `32`.